qemu/tests/cfg/hotplug_mem_negative.cfg

```diff
@@ -45,4 +45,4 @@
             addr_dimm_plug3 = 0x15000000000000
             keywords_plug3 = "can\'t add memory"
             addr_dimm_plug4 = 0x150800000
-            keywords_plug4 = "address range conflicts"
+            keywords_plug4 = "address range conflicts|can\'t add memory"
```

Comment:
IMO, `usable range for memory devices` is more suitable than `can't add memory`, how do you think ?

Reply:
I think "can't add memory" looks more like an error message than "usable range". So I prefer to keep it. Thanks.

Reply:
oh~   My thought it is `un-usable`......    Want to show it is the range related failure, sorry for that.